projects/i18n/languages/spanish/core.ts

```diff
@@ -2,22 +2,22 @@ import {LanguageCore} from '@taiga-ui/i18n/interfaces';
 
 export const TUI_SPANISH_LANGUAGE_CORE: LanguageCore = {
     months: [
-        'Enero',
-        'Febrero',
-        'Marzo',
-        'Abril',
-        'Mayo',
-        'Junio',
-        'Julio',
-        'Agosto',
-        'Septiembre',
-        'Octubre',
-        'Noviembre',
-        'Diciembre',
+        'enero',
+        'febrero',
+        'marzo',
+        'abril',
+        'mayo',
+        'junio',
+        'julio',
+        'agosto',
+        'septiembre',
+        'octubre',
+        'noviembre',
+        'diciembre',
     ],
     close: 'Cerrar',
     nothingFoundMessage: 'Nada encontrado',
-    defaultErrorMessage: 'El valor es válido',
+    defaultErrorMessage: 'El valor es inválido',
     spinTexts: ['Previo', 'Siguiente'],
-    shortWeekDays: ['Lun', 'Mar', 'Mie', 'Jue', 'Vie', 'Sáb', 'Dom'],
+    shortWeekDays: ['lun', 'mar', 'mié', 'jue', 'vie', 'sáb', 'dom'],
```

Comment:
Does it make more sense to use week days and months with small letter? They appear in calendars and look good with capital letter in Russian and English.

Reply:
To be honest, it is also looks better for me, if that's for the calendar component I don't see any problem using capitalization, but for formatting (for ex. '**next Monday**' = '**siguiente lunes**') the guideline is to use lowercase. 
I'm going to revert that change